js/src/table/gis_visualization.js

```diff
@@ -159,16 +160,15 @@ function initGISVisualization () {
     zoomAndPan();
 }
 
-function drawOpenLayerMap (openLayerCreate) {
+function drawOpenLayerMap () {
     $('#placeholder').hide();
     $('#openlayersmap').show();
     // Function doesn't work properly if #openlayersmap is hidden
-    if (!openLayerCreate) {
+    if (!map) {
         // Draws openStreetMap with openLayers
-        drawOpenLayers();
-        return 1;
+        map = drawOpenLayers();
     }
-    return 0;
+    return !!map;
```

Comment:
```suggestion
    return !!map;
```
Could this syntax be improved for readability ?

Reply:
It's actually not needed anymore, I removed it now.